Source/Shift/Classes/SplitTransition.swift

```diff
@@ -88,19 +102,132 @@ public class SplitTransition: NSObject {
     }()
 
     /**
+     * In an interactive transition, splitOffset
+     * determines the vertical distance of the initial
+     * split when the cell is first tapped
+     */
+    public var splitOffset: CGFloat = 0.0
+
+    /**
```

Comment:
I use the mighty [VVDocumenter](https://github.com/onevcat/VVDocumenter-Xcode) plugin.